debile/utils/xmlrpc.py

```diff
@@ -141,3 +142,13 @@ def get_proxy(config, auth_method):
                 ca_certs=xml.get('ca_certs', "/etc/ssl/certs/ca-certificates.crt")
             ), allow_none=True)
     return proxy
+
+
+def get_auth_method(args, config):
+    auth_method = args.auth_method
+
+    if 'xmlrpc' in config:
+        if 'auth_method' in config['xmlrpc']:
+            auth_method = config['xmlrpc']['auth_method']
```

Comment:
@clemux i did some unit tests without 'xmlrpc' section and did not raise any exception, you can see in tests/test_utils.py. Without this section does not raise an exception because it will access this variable only if exists in config, else it will bypass this code. Do you prefer handling this exception instead of this way?